### PR TITLE
Set minimum Python version for readthedocs

### PR DESCRIPTION
**Pull request by lpsinger**

### CODE REVIEW

File: {{ cookiecutter.package_name }}/.readthedocs.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ build:
   image: latest
 
 python:
-  version: 3.7
+  version: '{{ cookiecutter.minimum_python_version }}'
```
Comment (pllim) [inline {{ cookiecutter.package_name }}/.readthedocs.yml:7]: This won't work. RTD has changed the way they specify Python version for 3.8 or later.

https://github.com/astropy/astropy/blob/main/.readthedocs.yml

Comment (lpsinger) [reply {{ cookiecutter.package_name }}/.readthedocs.yml]: Good catch! Please take another look.

File: {{ cookiecutter.package_name }}/.readthedocs.yml [hunk 2]
```diff
@@ -2,9 +2,10 @@ version: 2
 
 build:
   image: latest
```
Comment (pllim) [inline {{ cookiecutter.package_name }}/.readthedocs.yml:4]:
```suggestion
  os: ubuntu-20.04
```

**Review by pllim — APPROVED** [review]
Thanks!

**Pull request merged**